lib/global_id/locator.rb

```diff
@@ -106,7 +106,9 @@ def use(app, locator = nil, &locator_block)
 
       private
         def locator_for(gid)
-          @locators.fetch(normalize_app(gid.app)) { default_locator }
+          @locators.fetch(normalize_app(gid.app)) do
+            gid.model_class.respond_to?(:unscoped) ? ACTIVE_RECORD_LOCATOR : DEFAULT_LOCATOR
```

Comment:
I don't like this at all. It's not duck-typing: it's [class detection](https://css-tricks.com/browser-detection-is-bad/).

We should either detect AR classes by looking at their class *or* stop pretending we've detected an AR instance, and call it an `UNSCOPED_LOCATOR` instead.